qiime2/sdk/tests/test_config.py

```diff
@@ -123,10 +137,8 @@ def test_mapping_only_config(self):
         self.assertEqual(dict_execution_contexts, self.tpool_expected)
 
     def test_mapping_from_dict(self):
```

Comment:
@Oddant1, what happens if you use the old format now (where the plugin name isn't specified). I think its worth testing that that throws an informative error message, just in case there are instances of that happening out in the wild (since this will be a backward incompatible change - but I think we bill it as a bug fix, since the previous behavior was that the same executor would be used for any `Pipelines` with the same name, regardless of what plugin they are in).